appointment/upload-documents/uploadDocumentsController.js

```diff
@@ -49,6 +48,27 @@ define('uploadDocumentsController', [], function() {
 		const ACCEPTABLE_FILE_TYPES = ['application/pdf', 'image/jpeg', 'image/png'];
 		const ACCEPTABLE_FILE_EXTENSIONS = ['.pdf', '.jpeg', '.jpg', '.png'];
 
+		$scope.storeConsent = () => {
```

Comment:
small nit: I'd consider calling this `submitConsent` instead.. I'm not a huge fan of `store`, but this is mostly personal preference